.github/workflows/heroku_deploy.yml

```diff
@@ -26,5 +27,6 @@ jobs:
             heroku builds:cache:purge --app=${{ env.APP_NAME }} --confirm=${{ env.APP_NAME }}
             heroku config:set BUILDPACK_RUN="pipenv run cookiecutter . --config-file $config_file --no-input -f" --app=${{ env.APP_NAME }}
           fi
+          cat $config_file
```

Comment:
a new echo and cat. Both just for logging insight?

Reply:
Yes please it was super helpful understanding when/why the mobile app was/wasn't building